Classes/Controller/GetFileController.php

```diff
@@ -15,31 +15,34 @@
  */
 
 /**
- * API to return METS or Attachement from Fedora
+ * API to return METS dissemination and Attachments from Fedora.
+ * Also renders METS XML for preview. Structure of the URIs totally
+ * depend on proper RealURL configuration.
  *
  * Example:
  *
  * 1. METS from Fedora
- *   http://localhost/api/qucosa:1234/mets
+ *   http://localhost/api/qucosa:1234/mets/
  *
  *   This always returns METS which is supplemented with additional information.
  *   The embedded MODS record is not the original MODS as it is stored in the
  *   repository datastream.
  *
  * 2. Attachment from Fedora
- *   http://localhost/api/qucosa:1234/attachment/ATT-0
+ *   http://localhost/api/qucosa:1234/attachment/ATT-0/
  *
- * 3. METS from Goobi.Publication (this extension)
- *   http://localhost/api/3/preview
+ * 3. METS from Kitodo.Publication (this extension)
+ *   http://localhost/api/3/preview/
  *
  *
  * @author    Alexander Bigga <alexander.bigga@slub-dresden.de>
+ * @author    Ralf Claussnitzer <ralf.claussnitzer@slub-dresden.de>
  */
```

Comment:
That is actually a good idea. I will see who is in charge for deciding this for Kitodo.Publication.